src/styl/index.styl

```diff
@@ -109,4 +116,4 @@ footer
 @import "imports/a11y"
 
 #content .item.visible
-	display initial
+	display initial
```

Comment:
You shouldn't have removed the last new line from this file, please lint your files as explained in the other comment